MC/run/examples/O2DPG_pp_minbias_one_tf_qc.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+#
+# A example workflow MC->RECO->AOD for a simple pp min bias 
+# production
+
+# make sure O2DPG + O2 is loaded
+[ ! "${O2DPG_ROOT}" ] && echo "Error: This needs O2DPG loaded" && exit 1
+[ ! "${O2_ROOT}" ] && echo "Error: This needs O2 loaded" && exit 1
```

Comment:
maybe one could add a check for `QC_ROOT`  or similar here

Reply:
Indeed, I will add this, thanks.